OMCompiler/Compiler/Script/CevalScript.mo

```diff
@@ -2964,14 +2964,18 @@ protected function unZipEncryptedPackageAndCheckFile
   output Boolean success;
   output String outFilename;
 protected
-  String workdir, s1, s2, s3, filename_1, filename1, filename2, filename3, filename4, str, str1, str2, str3, str4;
+  String workdir, s1, s2, s3, filename_1, filename1, filename2, filename3, filename4, str, str1, str2, str3, str4, cmd, cmdPrefix;
+  Boolean isWindows = Autoconf.os == "Windows_NT";
 algorithm
   success := false;
   outFilename := "";
   if (System.regularFileExists(filename)) then
     if (StringUtil.endsWith(filename, ".mol")) then
       workdir := if System.directoryExists(inWorkdir) then inWorkdir else System.pwd();
-      if (skipUnzip or 0 == System.systemCall("unzip -q -o -d \"" + workdir + "\" \"" +  filename + "\"")) then
+      // use ripunzip on Windows as is twice as fast
+      cmdPrefix := if isWindows then "ripunzip.exe -q unzip-file -d " else "unzip -q -o -d ";
+      cmd := cmd + "\"" + workdir + "\" \"" +  filename + "\"";
```

Comment:
Its suppose to be `cmd := cmdPrefix+ "\"" + workdir + "\" \"" + filename + "\"";`

Reply:
ouch :)